package.json

```diff
@@ -5,8 +5,9 @@
   "author": "",
   "license": "MIT",
   "scripts": {
-    "test": "jest 'unit'",
-    "integration-test": "jest 'integration'",
+    "test": "jest --runInBand",
```

Comment:
I think we don't need three test configurations - it would be enough to append `--runInBand` to unit-test and integration-test, wdyt?

Reply:
But how to run all tests then? always type the --runInBand if you want to do it locally?